examples/example-orca/src/main/java/io/grpc/examples/orca/CustomBackendMetricsLoadBalancerProvider.java

```diff
@@ -124,16 +124,24 @@ public LoadBalancer.PickResult pickSubchannel(LoadBalancer.PickSubchannelArgs ar
           return result;
         }
         // Installs ORCA per-query metrics reporting listener.
+        final OrcaPerRequestUtil.OrcaPerRequestReportListener orcaListener =
+            new OrcaPerRequestUtil.OrcaPerRequestReportListener() {
+          @Override
+          public void onLoadReport(MetricReport orcaLoadReport) {
+            System.out.println("Example load balancer received per-rpc metrics report:\n"
+                + orcaLoadReport);
+          }
+        };
+        if (result.getStreamTracerFactory() == null) {
```

Comment:
Note that this code demonstrates that we should probably improve the helpers (either by allowing null or by a more convenient method). But we can fix that later.